[Dewhale] implements #11

yzx396 opened the pull request:
[Dewhale] This PR implements #11, created by Dewhale.

yzx396 committed:
[Skip CI] Dewhale: init the PR

yzx396 committed:
[Dewhale] prompt:
{
  "prompt": "Create a web ui in react. On the left there is a form which has an input for FlagKey. On the right there is a textarea which shows the generated json according to the form input on the left. Whenever I make any change on the left, the right is updated in real time.\n",
  "images": []
}

yangzhaox commented:
[Dewhale]: This code snippet creates a React component with a form on the left side containing an input field for the `FlagKey`. On the right side, there is a textarea that displays the generated JSON in real-time as the input field is updated. The JSON structure includes the `flagKey`, a boolean `enabled` field, and an empty `variants` array. The `Textarea` is set to read-only since it's only for displaying the generated JSON.

yangzhaox commented:
Add a checkbox for State in the form

yzx396 committed:
[Dewhale] prompt:
{
  "prompt": "Create a web ui in react. On the left there is a form which has an input for FlagKey. On the right there is a textarea which shows the generated json according to the form input on the left. Whenever I make any change on the left, the right is updated in real time.\nAdd a checkbox for State in the form",
  "images": []
}
